src/test/java/com/revature/services/UserServiceTest.java

```diff
@@ -52,6 +54,19 @@ void testFindByCredentials() {
 		assertEquals(expected, actual);
 		verify(this.mockUserRepo, times(1)).findByEmailAndPassword(email, password);
 	}
+	
+	@Test
+	void testFindByCredentials_Failure_UserNotFound() {
+		String email = "user@coolmail.edu";
+		String password = "Alligator3";
+		given(this.mockUserRepo.findByEmailAndPassword(email, password)).willReturn(Optional.empty());
+		try {
+			this.uServ.findByCredentials(email, password);
+			fail("Expected an exception to be thrown");
+		} catch (Exception e) {
+			assertEquals(UserNotFoundException.class, e.getClass());
```

Comment:
You should specify which exception type is expected to be thrown. In this case, it's `UserNotFoundException`